specification/devcenter/resource-manager/Microsoft.DevCenter/preview/2024-06-01-preview/devcenter.json

```diff
@@ -0,0 +1,7236 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DevCenterNameParameter"
+          },
+          {
+            "$ref": "#/parameters/GalleryNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ImageNameParameter"
+          },
+          {
+            "$ref": "#/parameters/VersionNameParameter"
+          }
+        ],
+        "operationId": "ImageVersions_Get",
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ImageVersion"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Versions_Get": {
+            "$ref": "./examples/ImageVersions_Get.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DevCenter/projects/{projectName}/images": {
+      "get": {
+        "tags": [
+          "Images"
+        ],
+        "description": "Lists images for a project.",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "commonDefinitions.json#/parameters/ProjectNameParameter"
+          }
+        ],
+        "operationId": "Images_ListByProject",
+        "responses": {
+          "200": {
+            "description": "OK. The request has succeeded.",
+            "schema": {
+              "$ref": "#/definitions/ImageListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Images_ListByProject": {
+            "$ref": "./examples/Images_ListByProject.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DevCenter/projects/{projectName}/images/{imageName}": {
+      "get": {
```

Comment:
Why no PUT or DELETE methods? How do these get created?

Reply:
These resources are a curated, read-only proxy of images from the devcenter level, same as we currently do for devboxdefinitions: https://github.com/Azure/azure-rest-api-specs/pull/29364/files/e97f80356cee1a9342528ee5fbf35ace6cf385c3#diff-67d180abedf8d337fabf63aab93bc1e0567cca497533fc9a64d3ab740c458a67

Reply:
DevCenter admins attach a compute image gallery, and then can curate those images to the project level. Project admins can choose from various images at their project level to use when creating pools, and then users create devbox machines from those pools using the image.